docs/supported.md

```diff
@@ -502,55 +537,59 @@ One can stack font family, font weight, and font shape by using the `\textXX` v
+|$\%$ `\%`|$\cdots$ `\cdots`|$\LaTeX$ `\LaTeX`
+|$\#$ `\#`|$\ddots$ `\ddots`|$\TeX$ `\TeX`
+|$\&$ `\&`|$\ldots$ `\ldots`|$\nabla$ `\nabla`
+|$\_$ `\_`|$\vdots$ `\vdots`|$\infty$ `\infty`
+|$\text{\textunderscore}$ `\text{\textunderscore}`|$\dotsb$ `\dotsb`|$\infin$ `\infin`
+|$\text{--}$ `\text{--}`|$\dotsc$ `\dotsc`|$\checkmark$ `\checkmark`
+|$\text{\textendash}$ `\text{\textendash}`|$\dotsi$ `\dotsi`|$\dag$ `\dag`
+|$\text{---}$ `\text{---}`|$\dotsm$ `\dotsm`|$\dagger$ `\dagger`
+|$\text{\textemdash}$ `\text{\textemdash}`|$\dotso$ `\dotso`|$\text{\textdagger}$ `\text{\textdagger}`
+|$\text{\textasciitilde}$ `\text{\textasciitilde}`|$\sdot$ `\sdot`|$\ddag$ `\ddag`
+|$`$ <code>`</code>|$\mathellipsis$ `\mathellipsis`|$\ddagger$ `\ddagger`
+|$\text{\textquoteleft}$ `text{\textquoteleft}`|$\text{\textellipsis}$ `\text{\textellipsis}`|$\text{\textdaggerdbl}$ `\text{\textdaggerdbl}`
+|$\lq$ `\lq`|$\Box$ `\Box`|$\Dagger$ `\Dagger`
+|$\text{\textquoteright}$ `\text{\textquoteright}`|$\square$ `\square`|$\angle$ `\angle`
+|$\rq$ `\rq`|$\blacksquare$ `\blacksquare`|$\measuredangle$ `\measuredangle`
+|$\text{\textquotedblleft}$ `\text{\textquotedblleft}`|$\triangle$ `\triangle`|$\sphericalangle$ `\sphericalangle`
+|$"$ `"`|$\triangledown$ `\triangledown`|$\top$ `\top`
+|$\text{\textquotedblright}$ `\text{\textquotedblright}`|$\triangleleft$ `\triangleleft`|$\bot$ `\bot`
+|$\colon$ `\colon`|$\triangleright$ `\triangleright`|$\$$ `\$`
+|$\backprime$ `\backprime`|$\bigtriangledown$ `\bigtriangledown`|$\text{\textdollar}$ `\text{\textdollar}`
+|$\prime$ `\prime`|$\bigtriangleup$ `\bigtriangleup`|$\pounds$ `\pounds`
+|$\text{\textless}$ `\text{\textless}`|$\blacktriangle$ `\blacktriangle`|$\mathsterling$ `\mathsterling`
+|$\text{\textgreater}$ `\text{\textgreater}`|$\blacktriangledown$ `\blacktriangledown`|$\text{\textsterling}$ `\text{\textsterling}`
+|$\text{\textbar}$ `\text{\textbar}`|$\blacktriangleleft$ `\blacktriangleleft`|$\yen$ `\yen`
+|$\text{\textbardbl}$ `\text{\textbardbl}`|$\blacktriangleright$ `\blacktriangleright`|$\surd$ `\surd`
+|$\text{\textbraceleft}$ `\text{\textbraceleft}`|$\diamond$ `\diamond`|$\degree$ `\degree`
+|$\text{\textbraceright}$ `\text{\textbraceright}`|$\Diamond$ `\Diamond`|$\text{\textdegree}$ `\text{\textdegree}`
+|$\text{\P}$ `\text{\P}`|$\lozenge$ `\lozenge`|$\mho$ `\mho`
+|$\text{\S}$ `\text{\S}`|$\blacklozenge$ `\blacklozenge`|$\diagdown$ `\diagdown`
+|$\text{\sect}$ `\text{\sect}`|$\star$ `\star`|$\diagup$ `\diagup`
+|$\copyright$ `\copyright`|$\bigstar$ `\bigstar`|$\flat$ `\flat`
+|$\circledR$ `\circledR`|$\clubsuit$ `\clubsuit`|$\natural$ `\natural`
+|$\text{\textregistered}$ `\text{\textregistered}`|$\clubs$ `\clubs`|$\sharp$ `\sharp`
+|$\circledS$ `\circledS`|$\diamondsuit$ `\diamondsuit`|$\heartsuit$ `\heartsuit`
+|$\text{\textcircled a}$ `\text{\textcircled a}`|$\diamonds$ `\diamonds`|$\hearts$ `\hearts`
+|$\maltese$ `\maltese`|$\spadesuit$ `\spadesuit`|$\spades$ `\spades`
```

Comment:
FYI, there are screen widths where this does not render very well:

![image](https://user-images.githubusercontent.com/2218736/43960331-abe617a0-9c7f-11e8-87cb-4f0533b5aea1.png)

The cells are forced to be pretty wide, so I think the only solution would be to switch to two columns...  Eh, other tables look bad at this size too:

![image](https://user-images.githubusercontent.com/2218736/43960403-e6dfbf78-9c7f-11e8-8e38-52c9fc7dd65d.png)

Not sure we want to deal with this now.

Reply:
>  there are screen widths where this does not render very well:

Currently, both sidebars disappear at the same quite narrow break point. I'll see if I can make just one sidebar disappear at an intermediate break point.  

I'd rather not make the tables more narrow than they already are. If users have to scroll up and down a lot to see just one topic, then I'd prefer to get rid of this page and put a topic filter on the Support Table page.